bigbluebutton-html5/imports/ui/components/closed-captions/component.jsx

```diff
@@ -39,22 +41,28 @@ export default class ClosedCaptions extends React.Component {
   componentDidUpdate() {
     if (this.shouldScrollBottom) {
       const { ccScrollArea } = this.refs;
-      var node = findDOMNode(ccScrollArea);
-      node.scrollTop = node.scrollHeight
+      const node = findDOMNode(ccScrollArea);
+      node.scrollTop = node.scrollHeight;
     }
   }
 
   render() {
+    const {
+      locale,
+      captions,
+      backgroundColor,
+    } = this.props;
+
     return (
       <div disabled className={styles.ccbox}>
         <div className={styles.title}>
-          <p> {this.props.locale ? this.props.locale : 'Locale is not selected'} </p>
+          <p> {locale ? locale : 'Locale is not selected'} </p>
         </div>
         <div
           ref="ccScrollArea"
```

Comment:
I think I will leave it for now and will replace all of them when we do the transition